Application/ControlMsrE2/ControlMsrE2.c

```diff
@@ -0,0 +1,210 @@
+/** @file
+ Control CFGLock BIOS Option
+ 
+ Copyright (c) 2020, Brumbaer. All rights reserved.<BR>
+ This program and the accompanying materials
+ are licensed and made available under the terms and conditions of the BSD License
+ which accompanies this distribution.  The full text of the license may be found at
+ http://opensource.org/licenses/bsd-license.php
+ 
+ THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+ WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+ 
+ **/
+#include "ControlMsrE2.h"
+
+#define CONTEXTS_MAX 8
+
+EFI_GUID FormSetupClassGuid = EFI_HII_PLATFORM_SETUP_FORMSET_GUID;
+
+EFI_STATUS
+EFIAPI
+UefiMain (
+          IN EFI_HANDLE        ImageHandle,
+          IN EFI_SYSTEM_TABLE  *SystemTable
+          )
+{
+    EFI_HII_HANDLE * handles;
+    UINT32           OptionsCount;
+    UINT32           ContextsCount;
+    OneOfContext     Contexts[CONTEXTS_MAX];
+    
+    EFI_STATUS Status = InterpretArguments();
+    
+    if (!EFI_ERROR(Status)) {
+        Status = VerifyMSRE2 (ImageHandle, SystemTable);
+    }
+    
+    if (!EFI_ERROR(Status)) {
+        Print(L"\nBIOS Options:\n");
+
+        EFI_STRING SearchString = AllocateStrFromAscii ("cfg");
```

Comment:
You could use `AsciiStrCopyToUnicode` from OcStringLib.h here.

Reply:
I can do this.